recipes/sentry-native/all/conanfile.py

```diff
@@ -130,6 +130,7 @@ def validate(self):
             raise ConanInvalidConfiguration("This version doesn't support ARM compilation")
 
     def build_requirements(self):
+        self.build_requires("cmake/3.22.0")
```

Comment:
```suggestion
        if tools.Version(self.version) >= "0.4.14":
            self.build_requires("cmake/3.22.0")
```

see https://github.com/conan-io/conan-center-index/pull/9200#discussion_r798450723